dotnet-client-libraries/WireMock.NET/WireMockNet/PlanetsService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Net;
+using System.Text.Json;
+
+namespace WireMockNet
+{
+    public class PlanetsService
+    {
+        private readonly HttpClient _httpClient;
+
+        public PlanetsService(HttpClient httpClient)
+        {
+            _httpClient = httpClient ??
```

Comment:
Because you have enabled nullables for the project (which is the default with new projects starting in .NET 6.0), the HttpClient is attributed to never be null so the check is unnecessary, and most likely the IDE would even be showing you that httpClient cannot be null here.

Reply:
Resolved in a separate PR https://github.com/CodeMazeBlog/CodeMazeGuides/pull/1182